src/image-copy/azext_imagecopy/create_target.py

```diff
@@ -21,10 +21,10 @@ def create_target_image(location, transient_resource_group_name, source_type, so
 
     random_string = get_random_string(STORAGE_ACCOUNT_NAME_LENGTH - len(location))
 
-    # create the target storage account
+    # create the target storage account. storage account name must be lowercase.
     logger.warn(
         "%s - Creating target storage account (can be slow sometimes)", location)
-    target_storage_account_name = location + random_string
+    target_storage_account_name = location.lower() + random_string.lower()
```

Comment:
random_string is already lowercase only
```suggestion
    target_storage_account_name = location.lower() + random_string
```